wsic/utils.py

```diff
@@ -312,29 +297,23 @@ def wrap_index(
     reverse: bool = True,
 ) -> Tuple[Tuple[int, ...], int]:
     """Wrap an index to the shape of an array.
-
     Args:
         index (Tuple[int, ...]):
             The index to wrap.
         shape (Tuple[int, ...]):
             The shape of the array.
         reverse (bool):
             If True, wrap the index to the opposite end of the array.
-
     Returns:
         Tuple[Tuple[int, ...], int]:
             The wrapped index and any overflow.
-
     Examples:
         >>> wrap_index((0, 3), (3, 3))
         ((1, 0), 0)
-
         >>> wrap_index((1, 4), (3, 3))
         ((2, 1), 0)
-
         >>> wrap_index((3, 1), (3, 3), reverse=False)
         ((0, 2), 0)
-
     """
```

Comment:
Please leave spacing. Here the examples have all become merged into one which is not correct.